profile-service/src/main/resources/db/migration/V9.14__update_reval_role_name.sql

```diff
@@ -0,0 +1,45 @@
+-- add the new roles
+
+INSERT INTO `Role` (`name`)
+VALUES
+      ('RevalApprover'),
+      ('RevalAdmin'),
+      ('RevalObserver')
+ON DUPLICATE KEY UPDATE `name` = `name`;
+
+-- update the user role with the new roles
+
+UPDATE `UserRole`
+SET `roleName` = 'RevalApprover'
+WHERE `roleName`='RevalTISAdmin1';
+
+UPDATE `UserRole`
+SET `roleName` = 'RevalAdmin'
+WHERE `roleName`='RevalTISAdmin2';
+
+UPDATE `UserRole`
+SET `roleName` = 'RevalObserver'
+WHERE `roleName`='RevalTISAdmin3';
+
+UPDATE `UserRole`
+SET `roleName` = 'HEE User Admin'
+WHERE `roleName`='RevalSiteAdmin';
```

Comment:
Out of interest, if the trainee already got a 'HEE User Admin' role in the UserRole table, will he get a duplicate role for it after update? Not sure if Flyway and H2 support something like `WHERE NOT EXISTS`?

Reply:
Further to the discussion with Ade, we confirm that we can simply `DELETE` the role `RevalSiteAdmin`